overlay/files/etc/dracut.conf.d/99-kairos.conf

```diff
@@ -0,0 +1 @@
+install_items+=" /etc/systemd/network/20-dhcp.network "
```

Comment:
This is best to go as a package so our initrds are aligned as well, but best for a follow-up